Examples/Examples-iOS/IGListKitExamples/ViewControllers/MixedDataViewController.swift

```diff
@@ -23,7 +23,8 @@ final class MixedDataViewController: UIViewController, ListAdapterDataSource {
     let collectionView = UICollectionView(frame: .zero, collectionViewLayout: UICollectionViewFlowLayout())
 
     let data: [Any] = [
```

Comment:
@rnystrom I have given it a try I had to make a code change to make the filtering work again. As now it didn't wanted to filter the strings straight away. After some struggling to push my commit to my branch I think it made it up here.